src/containers/NibrsContainer.js

```diff
@@ -117,8 +117,7 @@ const NibrsContainer = ({
       {content}
       {isReady &&
         <div className="serif italic fs-12">
-          Source: Reported {nibrsTerm} data from {placeDisplay},{' '}
-          {nibrsFirstYear}–{until}.
+          Source: Reported {nibrsTerm} data from {placeDisplay}.
```

Comment:
Ya, I see your point. This reduces the complexity and the year range is always specified in the header. In the case of the NIBRS container it also contains a sentence about when the entity first started reporting NIBRS.